src/style.css

```diff
@@ -0,0 +1,19 @@
+.height {
+  height: 100px;
+}
+
+.first {
+  background-color: red;
+}
+
+.second {
+  background-color: green;
+}
+
+.third {
+  background-color: blue;
+}
+
+body {
```

Comment:
better to write common style on the top of the document.
move `body` to the top.